titus-api/src/main/java/com/netflix/titus/api/jobactivity/service/JobActivityPublisherService.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.api.jobactivity.service;
+
+/**
+ * This interface represents a service that monitors job activity events/changes
+ * and propagates them to a publisher store.
+ */
+public interface JobActivityPublisherService {
+
+    void activate();
```

Comment:
Is this interface needed? It has no clients. The activation process methods should not be exposed here. They are part of the implementation details.